msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -409,6 +417,10 @@ public Boolean authorizationInCurrentTask() {
         return isAuthorizationInCurrentTask;
     }
 
+    public Boolean isWebauthnCapable() {
+        return webauthnCapable != null && webauthnCapable;
```

Comment:
Could probably just do `webauthnCapable == Boolean.TRUE`

Reply:
Done: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1895/commits/0945c6818094fa5624bcfc7129deeb9249ef4d55